combat-trainer.lic

```diff
@@ -873,7 +872,10 @@ class LootProcess
   end
 
   def check_skinning(mob_noun, game_state)
+    already_arranged = false
     if @dissect && game_state.dissectable?(mob_noun)
+      arrange_mob(mob_noun, game_state) if @always_arrange
```

Comment:
I think this _could_ change behavior, since we aren't guaranteed to dissect at this point, right?

Reply:
I don't think so, because either way, we are in the dissect code, and even if we arrange here, and don't dissect, we just do it early for the skinning routine further down, which won't arrange again.